gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -606,6 +606,7 @@ nnsconf_get_custom_value_string (const gchar * group, const gchar * key)
 
       if (key_file == NULL) {
         ml_loge ("Failed to create key file object for custom config");
+        g_free (hashkey);
```

Comment:
How about replacing line 583 with the following and removing all ```g_free(hashkey)``` ?
```
g_autofree gchar *hashkey = g_strdup_printf ("[%s]%s", group, key);
```

And , line 620
```
g_hash_table_insert (custom_table, g_steal_pointer (&hashkey), value);
```

Reply:
Thanks, I updated this PR. Please see the changes again.

Reply:
I checked everything. :+1: